unicodetools/src/main/java/org/unicode/props/IndexUnicodeProperties.java

```diff
@@ -228,7 +228,17 @@ public UnicodeMap<Double> loadDouble(UcdProperty prop2) {
                             Integer.parseInt(value.substring(0, pos))
                                     / (double) Integer.parseInt(value.substring(pos + 1));
                 } else {
-                    convertedValue = Double.parseDouble(value);
+                    String v = value;
+                    if (prop2 == UcdProperty.kPrimaryNumeric
```

Comment:
This is ok for now. The items that are really sets or lists have a status indicated in the data files, so that they can be correctly parsed, and the values are maintained.

Reply:
I should say, in the _metadata_ file Multivalued.txt

Reply:
Captured in https://github.com/unicode-org/unicodetools/issues/356 -- let's fix that in a follow-up PR.
Would you like to do this?
Or teach someone else?